### ISSUE TITLE
[Suggestion][B15] I miss the old research, tbh.

### ISSUE THREAD

**Issue by applebaps** [Azanor/thaumcraft-suggestions#296]
Maybe once the reimagined, simpler research is more fully-featured, I'll like it more, but I really enjoyed the immersive quality of connecting the different aspects on the hex grid.  It really conveyed a feeling that you as the player were achieving some new synthesis of ideas, or doing actual magical research, just by virtue of how it worked.  The new research game feels explicitly "gamey" with its card drawing, element of choice, and clear risk/reward structure.  This seems to be working oddly at cross-purposes with the rewritten first person perspective and immersive tone of the revamped Thaumonomicon and the new start process and such.

I know that's not very constructive on its own ("change is bad! don't change!" lol), so to soften the blow, here's a suggestion for how the new research game could be improved.

What if the cards you drew weren't themed to let you clearly pick your research topic, but instead were themed around the Aspects, almost like a tarot deck that you're drawing from to gain new insight?  Instead of having the cards side by side, pick this or that, the player could experiment with positioning multiple Aspect Cards in a kind of tarot-inspired layout that would in turn connect to new concepts.  What I'm envisioning would be kind of like a compromise between the aspect hex grid and the current revamped research.  You could have spreads to invoke each category of research; if the player wants to research Alchemy, they could lay out aspects in a certain way, or to research Auromancy, another kind of spread.

From there, though, placing the Aspect cards in certain ways within the spread could unlock new insights.  So for example, let's say the player wants to figure out some way to reverse the taint (a common problem in the TC6 beta).  They're looking for an alchemical solution, so they use that spread.  For the sake of argument, let's say it's 3 cards in a row, something simple.

They know they want to know about the source of the taint, so they lay out an Ace of Vitium (or something) in slot 1.  Because they're interested in magically cleansing it, maybe they want the Jack of Auram in slot 2.  Then, because it'll be something organic or planted (like, say, a purifying bloom), an Herba card goes in slot 3.  This then unlocks either the existing research choice draws (maybe with some rebranding or at least like some aspect drawings on the cards or SOMETHING to immerse a bit more), or just outright gives theoretical knowledge points.

This isn't a fully-formed idea, granted.  Take it or leave it! :)  I'd be willing to help pitch in with balancing and refining the concept in a dialogue, if you like.

**Comment by angeltxilon** [comment]
I prefer the current ones. The old research system was extremely slow and required a lot of time for obtain aspects (taking in consideration that the knowledge obtained was usually not very powerful nor game-changing).

The thing that I really miss, is the old system of 6 primal types of vis, and nodes from thaumcraft 4; and runic portals (telporting portals with special rendering and animation) from thaumcraft 2 or 3.

**Comment by Geethebluesky** [comment]
This is interesting to see different perspectives. I'm in the camp that finds the hex minigame to be much too gamey compared to the current setup... it feels unrelated to the actual discoveries made, whereas the current cards provide more story and background into the process, along with slowing people down a bit from discovering everything all at once with the use of consumed or held items (since it makes no sense for such a complex process to take an average of minutes or less after memorizing how aspects combine).

I like the idea of combining the cards in a more deliberate way though. But since these cards would physically combine instead of just being progress markers like the current cards are, I really think the player should have to **create** these aspect cards themselves, not just have them given by the mod. 

Maybe this can bridge another gap: makes it mandatory for the player to scan every item in the world, and receive either a card straight out (possible but probably clunky) or gain the knowledge to craft the card from the observations made.

**Comment by jvcmarcenes** [comment]
Yeah, but the new research system, at least I believe, is meant to make the player leave the research table (in previous versions it was all very related to the research table, most of the gameplay was getting the aspects you needed (which was terrible), and then laying then in the hex), know the player need to go after crystals and items and essentia phials, that makes researching more dinamic and actually makes it feels like you're studying stuff
maybe there could be a system based around your ideas of cards, I quite liked it, but then it woudn't be much of researching...
alright, I got and idea, why not another area of magic? maybe like transmutation, maybe you can take a special item, an d then put them in a table and have to create a card grid toturn it into another, if you fail you can loose the item, or get some bad stuff?
a special crafting device based around this cards (maybe make the cards like a item the player has to find in chests and loot and mob drops, maybe even add random loot aroundthe world to make finding this easier... since there isn't alot of free laying chests in the game, well, in the magical forests you do have a lot of those cobwebs greatwood trees that contain loot)
Maybe a special Villager that can trade for this cards, and pechs could give you special cards that make rarer items
Yeah, I'm excited with this idea, ifazanor doesn't get to do this I'll try coding this myself, I'm trying to get into mod making since I have coding experience and free time
This woudn't change the theorycrafting system, but would add a whoçe new layer... maybe cards could also be used for summoning, yeah... I have a gigantic brainstorm rn
Anyone thinks that this would be awesome as well??

**Comment by Geethebluesky** [comment]
So with your idea, magical theory is "out there" in the world and waiting to be reconstituted from parts (items, blocks, etc.) into ideas, am I understanding that correctly?

Because that sounds freaking awesome.

**Comment by jvcmarcenes** [comment]
yeah @Geethebluesky, Just made a Suggestion Issue called cartomancy explaining more of this idea
